src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -60,6 +60,9 @@ import {
   sponsorDischargePapersConfig,
   blankSchema,
   acceptableFiles,
+  sponsorCasualtyUploadUiSchema,
+  sponsorDisabilityRatingUploadUiSchema,
+  sponsorDischargePapersUploadUiSchema,
 } from '../components/Sponsor/sponsorFileUploads';
```

Comment:
These files flagged as missing by linter are introduced in https://github.com/department-of-veterans-affairs/vets-website/pull/28835